tests/components/wunderground/test_sensor.py

```diff
@@ -90,7 +95,7 @@ async def test_sensor(hass, aioclient_mock):
     state = hass.states.get("sensor.pws_weather")
     assert state.state == "Clear"
     assert state.name == "Weather Summary"
-    assert "unit_of_measurement" not in state.attributes
+    assert ATTR_UNIT_OF_MEASUREMENT not in state.attributes
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6212145)